front/app/modules/free/user_confirmation/citizen/components/ConfirmationSignupStep.tsx

```diff
@@ -127,8 +127,7 @@ const ConfirmationSignupStep = ({ onCompleted, onData, step }: Props) => {
       key: CONFIRMATION_STEP_NAME,
       position: 4,
       stepDescriptionMessage: messages.confirmYourAccount,
-      isEnabled: (authUser, __, { emailSignUpSelected }) => {
-        if (emailSignUpSelected) return true;
+      isEnabled: (authUser) => {
         return isActive(authUser);
```

Comment:
In an email sign-up, isActive(authUser) would also return true. It checks if the user still needs to confirm their email.